graphql/config_resolver.go

```diff
@@ -2,11 +2,18 @@ package graphql
 
 import (
 	"context"
+	"fmt"
 	"sort"
 
+	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/rest/model"
 )
 
+// BannerTheme is the resolver for the bannerTheme field.
+func (r *adminSettingsResolver) BannerTheme(ctx context.Context, obj *model.APIAdminSettings) (*evergreen.BannerTheme, error) {
+	panic(fmt.Errorf("not implemented: BannerTheme - bannerTheme"))
+}
+
```

Comment:
you'll have to fill this out; you could do something similar to the resolvers in `distro_settings_resolver.go` which convert values to enums

this is just because the `APIAdminSettings` types `BannerTheme` as a `*string`... not sure why but we can leave that alone for now.